tools/beerocks_analyzer/beerocks_analyzer_widget.py

```diff
@@ -488,7 +504,7 @@ def addRemoveNewFigSubplots(self, ar_t, ar_v, target_lable, is_add, entity, enti
                         else:
                             color = self.getLineColor(entity, entity_num)
                             if color == None:
-                                PRINTF("Error, color == None, param_n=%s\n", param_n)
+                                self.logger.error("Error, color == None, param_n={}".format(param_n))
                                 return
                             marker = self.marker[0]
                             if self.fig_subplot_nums2marker.has_key((f,p)):
```

Comment:
There are still some instances of has_key let in beerocks_analyzer_widget.py.

(nitpick)